pkg/comp-functions/functions/vshnkeycloak/deploy.go

```diff
@@ -412,6 +508,26 @@ func toYAML(obj any) (string, error) {
 	return string(yamlBytes), err
 }
 
+func copyCustomImagePullSecret(svc *runtime.ServiceRuntime, comp *vshnv1.VSHNKeycloak) error {
+	secretClaim := &corev1.Secret{}
+
+	err := svc.SetDesiredKubeObserveObject(secretClaim, comp.GetName()+"-custom-image-pull-secret-observer")
+	if err != nil {
+		return err
+	}
```

Comment:
You don't seem to get this observer ever again... So it's probably not necessary?

Reply:
Fixed